core/src/test/java/nl/nn/adapterframework/pipes/XmlSwitchTest.java

```diff
@@ -92,7 +93,7 @@ public void xPathFromParameterWildCardNamespaced() throws Exception {
 		inputParameter.setType("domdoc");
 		pipe.addParameter(inputParameter);
 		session.put("sessionKey", input);
-		pipe.setXsltVersion(1);
```

Comment:
Why did you remove this? did the above change break this?

Reply:
yes